docs/hugo.toml

```
@@ -1,6 +1,6 @@
 baseURL = "/"
 title = "Tetragon - eBPF-based Security Observability and Runtime Enforcement"
-description = "eBPF-based Security Observability and Runtime Enforcement"
+description = "Tetragon is a low overhead, flexible, and Kubernetes native security observability tool that applies policy and filtering directly in the kernel"
```

```suggestion
description = "Tetragon is a low overhead, flexible, and Kubernetes native security observability tool that applies policy and filtering directly in the kernel using eBPF"
```

Would it make sense to keep eBPF in the description?